projects/packages/publicize/src/class-publicize.php

```diff
@@ -54,21 +54,6 @@ public function add_disconnect_notice() {
 		add_action( 'admin_notices', array( $this, 'display_disconnected' ) );
 	}
 
-	/**
-	 * Whether to use the v1 admin UI.
-	 */
-	public function use_admin_ui_v1(): bool {
```

Comment:
I don't think it will as the signature is the same and we're moving it to the parent class, but is there any possibility that moving this method will cause an issue?

Reply:
Do you mean that we should remove the return type here?